client/product_details/ProductInfo.jsx

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 
 const ProductInfo = (props) => {
   if (props.info && props.selectedStyle) {
+    const priceState = props.selectedStyle.sale_price === null ? 'regular' : 'sale';
     return (
       <div id="productInfo">
         <div id="productCategory">{(props.info.category).toUpperCase()}</div>
         <div id="productTitle">{props.info.name}</div>
-        <div id="productPrice">${props.info.default_price}</div>
+        {priceState === 'sale' ? <span id="productSalePrice">${props.selectedStyle.sale_price}</span> : null}
```

Comment:
Slick! Quick question, why use a span instead of a div for these? I'm curious since I'm not sure of the pros/cons of using them

Reply:
I want them to be side by side so a span or a works better than a div -- the div generally fills the width of the container and would have to be manipulated via CSS